.pre-commit-config.yaml

```diff
@@ -55,6 +55,27 @@ repos:
       minimum_pre_commit_version: 2.8.0
       name: Check notebooks have watermark (see Jupyter style guide from PyMC docs)
       types: [jupyter]
+    - id: no-internal-links
+      name: Check no internal links are in the docs
+      description: >-
+        'A quick check to prevent urls pointing to pymc docs'
+      files: ^examples/
+      exclude: >
```

Comment:
The 404.md file should also be excluded. And for now I would exclude all the notebooks that fail the check too so we can merge the PR and then work on fixing the issues

Reply:
`404.md` added in https://github.com/pymc-devs/pymc-examples/pull/311/commits/22e83a6e6c6f1b3f7e8ac70c767422268420b6af

Reply:
I added all the notebooks which were failing and I also had to add `index.md` to that the tests pass.